apps/ledger-live-mobile/src/components/CustomImageDeviceAction.tsx

```diff
@@ -114,6 +114,7 @@ const CustomImageDeviceAction: React.FC<Props & { remountMe: () => void }> = ({
               error,
               device,
               ...(isRefusedOnStaxError ? { Icon: Icons.Warning, iconColor: "warning.c60" } : {}),
+              hasExportLogButton: false,
```

Comment:
Should the value be: `!isRefusedOnStaxError` ? Or we don't want the export log button for any kind of error as long as `isError` is true ?